fastparquet/core.py

```diff
@@ -145,7 +154,7 @@ def read_dictionary_page(file_obj, schema_helper, page_header, column_metadata):
 
 
 def read_col(column, schema_helper, infile, use_cat=False,
-             grab_dict=False):
+             grab_dict=False, selfmade=False):
```

Comment:
This term selfmade is not immediately clear to me.  Does this mean made by the fastparquet project?